prepare_database.go

```diff
@@ -55,6 +55,11 @@ func defaultInitDatabase(binaryExtractLocation, runtimePath, pgDataDir, runAsUse
 			return err
 		}
 
+		err = chownR(binaryExtractLocation, runAsUser)
+		if err != nil {
```

Comment:
nit: can be one-lined

Reply:
although it won't be visually similar to the rest of the code in this func so whatever